plugin/src/main/java/org/opensearch/ml/task/MLPredictTaskRunner.java

```diff
@@ -405,6 +422,10 @@ private void runPredict(
                                                     remoteJob
                                                 );
 
+                                                if (!clusterService.state().metadata().indices().containsKey(TASK_POLLING_JOB_INDEX)) {
+                                                    mlTaskManager.startTaskPollingJob();
+                                                }
+
```

Comment:
This starting Polling job doesn't need anything from a specific task, so we can start this polling job in the MachineLearningPlugin when the ml-commons is loaded?

Reply:
Yes, but it seemed better to start if with the first batch predict request. If user is not using batch prediction on their cluster at all, it does not seem necessary to initiate the job. So, we will initiate the job with the first batch predict request to the cluster.